### ISSUE TITLE
Issues with Citations and Pandoc

### ISSUE THREAD

**Issue by maurerle** [jgm/pandoc#8136]
**Explain the problem.**
I am using a markdown file with bibliography and convert to a revealjs presentation:
`pandoc -s --bibliography biblio.bib --citeproc CITATIONS -t revealjs -o example24a.html`

as taken from the examples (except for revealjs).

This presents some issues:
1. references are not split across multiple pages if needed
    * of course this is not the case with normal bullet points either, but you can't edit this for references?
    * this should also be fixed for beamer output
2. links are not clickable, even with `link-citations: true` 
    * this works properly with beamer output, but not with revealjs 

I would suggest to create an option of `citation-presentation-count` to have only 3 references per page?
Any other suggestion? Currently the references are not really usable for presentations (beamer and revealjs) with more than 3 sources listed.
But I'd really love to have this integrated.

**Pandoc version?**
What version of pandoc are you using, on what OS?

pandoc 2.18 on Debian 11

**Comment by maurerle** [comment]
I solved this issue for my use case by adding the following to my style.css for revealjs:

```css
.references {
  font-size: 0.5em;
  text-align: left;
}
```

Might be helpful for others.
Yet a more general solution is still needed, as this only fits my 10ish sources and is not very generic

**Comment by jgm** [comment]
The solution for beamer is to put the class `allowframebreaks` on your references slide:

```
# References {#refs .allowframebreaks}

```

**Comment by jgm** [comment]
I don't know if there is anything like `allowframebreaks` for reveal.js, but if so that could be a solution to 1.

I'm not sure about 2 -- link-citations ought to work in reveal.js.  Looks like there's a bug, we're getting links that look like
```
href="#/ref-foo_1938"
```
instead of
```
href="#ref-foo_1938"
```

**Comment by jgm** [comment]
See https://revealjs.com/links/
However, this doesn't seem to work in my tests. Perhaps it only works if the link is to an identifier on a section?

**Comment by jgm** [comment]
This seems to be an odd limitation in reveal.js -- you can link to a section (slide) but not to something within one.  Pandoc uses a generic method when you select `link-citations`, and this links to the citation itself, not the references slide, so this won't work with reveal.js.

**Comment by jgm** [comment]
A more general CSS solution for the references slide is
```
#refs {
    height: 800px;
    overflow-y: auto !important;
}
```
which will give you a scroll bar.

**Comment by metaspace** [comment]
@maurerle did you ever find a way to paginate the references?

**Comment by maurerle** [comment]
No, I either use the scrollbar or make the text on the citation slide smaller.

But at least linking to rdferences works properly in reveal.js :)

**Comment by KorigamiK** [comment]
> No, I either use the scrollbar or make the text on the citation slide smaller.
> 
> But at least linking to references now works properly in reveal.js :)

can you tell what option allows for linking to references slide in the citations
